src/Controls/src/Core/View.cs

```diff
@@ -147,11 +150,33 @@ public IList<IGestureRecognizer> GestureRecognizers
 			get { return _gestureRecognizers; }
 		}
 
+		bool IsPointerOver = false;
+		PointerGestureRecognizer _recognizerForPointerOverState = new PointerGestureRecognizer();
```

Comment:
- Move these properties to the top of the file
- I wouldn't instantiate this here. Instantiate it inside `CompositeGesturesRecognizers`

_recognizerForPointerOverState ??= new PointerGestureRecognizer();